README.md

```diff
@@ -13,44 +13,58 @@ Static data that other applications (e.g., monitoring and visualization services
 - Templates
 - Beacons
 
-## `/data/chains.json`
+Subdirectories represent different protocols, namely : `rrp-psp` and `proto-psp`
 
-A JSON file that contains data related to the chains. These chains will not necessarily have official support. Each
-chain has the following specified at minimum:
+## Common files
+
+#### `/data/chains.json`
+
+A JSON file that contains data related to operational chains. These chains will not necessarily have official support.
+Each chain's contracts are versioned. Contract deployments have the following specified at minimum:
 
 - `AccessControlRegistry`: AccessControlRegistry contract address
 - `AirnodeRrp`: AirnodeRrp contract address
+
+and optionally:
+
 - `RrpBeaconServer`: RrpBeaconServer contract address
+- `DapiServer`: The DapiServer contract address
+
+### `/data/api3Metadata.json`
+
+A file containing metadata related to api3 operations around an on-chain resource, mostly used for backup services.
+
+### `/data/documentationMetadata.json`
```

Comment:
Is this referring to https://github.com/api3dao/operations/blob/v1-schema/data/documentation.json